nowcasting_forecast/app.py

```diff
@@ -124,7 +124,9 @@ def run(
                     )
                 elif model_name == "cnn":
                     dataloader = get_cnn_data_loader(
-                        src_path=f"{temporary_dir}/live", tmp_path=f"{temporary_dir}/live"
+                        src_path=f"{temporary_dir}/live",
+                        tmp_path=f"{temporary_dir}/live",
+                        batch_save_dir=batch_save_dir,
```

Comment:
do you need to add this to get_cnn_data_loader too?

Reply:
Done! Just had forgotten to push that change